components/camel-linkedin/camel-linkedin-api/src/main/java/org/apache/camel/component/linkedin/api/LinkedInOAuthRequestFilter.java

```diff
@@ -89,8 +89,11 @@ public LinkedInOAuthRequestFilter(OAuthParams oAuthParams, Map<String, Object> h
                                       boolean lazyAuth, String[] enabledProtocols) {
 
         this.oAuthParams = oAuthParams;
-        this.oAuthToken = null;
-
+        if(oAuthParams.getSecureStorage() != null) {
```

Comment:
There is a little code formatting problem here. If you run `mvn compile -P sourcecheck` you can get a report of the problem.

Reply:
@davsclaus Thank you for noticing it. Fix is commited to the PR.
Just from my curiosity, is possible that I'm running it wrongly? I've checked mvn compile -P sourcecheck with successful result even if there was obvious error:
[INFO] Camel :: LinkedIn :: Parent ........................ SUCCESS [ 20.998 s]
[INFO] Camel :: LinkedIn :: API ........................... SUCCESS [  4.024 s]
[INFO] Camel :: LinkedIn :: Component ..................... SUCCESS [01:20 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:46 min
[INFO] Finished at: 2019-03-20T08:42:45+01:00
[INFO] Final Memory: 49M/165M
[INFO] ------------------------------------------------------------------------
[jondruse@ibm-p8-rhevm-01-fsp camel-linkedin]$ mvn compile -P sourcecheck

Reply:
This should be because camel-linkedin is splitted in api and component, if you run the same command inside the component or api, you should see the error

Reply:
@oscerd Thanks. I've tried to run it in api module where there is one checkstyle error and my result is:
[INFO] --- maven-checkstyle-plugin:3.0.0:checkstyle (default-cli) @ camel-linkedin-api ---
[INFO] Starting audit...
[ERROR] /home/jondruse/git/fuse/camel/components/camel-linkedin/camel-linkedin-api/src/main/java/org/apache/camel/component/linkedin/api/LinkedInOAuthRequestFilter.java:92:11: 'if' is not followed by whitespace. [WhitespaceAround]
Audit done.
[INFO] There is 1 error reported by Checkstyle 7.6.1 with camel-checkstyle.xml ruleset.
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS

I can see checkstyle error in console, but build ends successfully. Is this  correct behavior?

Reply:
yes, we don't consider this something to mark the build as failed, in the past we try to enforce that, but it was too much

Reply:
@oscerd Now it makes sense to me. I haven't checked console, as result was success. So I've ignored this checkstyle problem. Now I know what to do in the next case. Thank you once again.